src/pages/index.tsx

```diff
@@ -36,19 +41,26 @@ const Index: NextPage = () => {
 					content="Ledger WalletConnect"
 				/>
 			</Head>
-			{isMounted ? (
-				<WalletApiClientProvider>
-					{(accounts, userId, walletInfo) => (
-						<WalletConnect
-							initialMode={initialMode}
-							initialURI={uri}
-							accounts={accounts}
-							userId={userId}
-							walletInfo={walletInfo}
-						/>
-					)}
-				</WalletApiClientProvider>
-			) : null}
+
+			{isApplicationDisabled ? (
+				<ApplicationDisabled />
+			) : (
+				<>
```

Comment:
You can directly put the `isMounted ? ...` after the `) : ` above instead of putting it in a fragment